README.md

```diff
@@ -86,6 +86,67 @@ def play(self):
 			return self.energy
 ```
 
+and add the following lines right under `time.sleep(self.latency)` on line 27:
+
+```python
+if debug:
+	next = input()
+	if next:
+		return 'Next'
+```
+
+### Edit `Map.py`
+Lastly, in `Map.py`, add:
+
+```python
+self.overlap = [str(i) for i in range(10)]
+```
+
+on line 19, right below:
+
+```python
+self.botPosition = self.checkpoint[:] #bot always starts at the checkpoint` on line 18.
+```
+
+Then, change the signature of the `moveRobot()` method on line 58, from:
+
+```python
+def moveRobot(self, move)
+```
+
+to:
+
+```python
+def moveRobot(self, move, debug=False)
+```
+
+and change the lines after:
```

Comment:
I like the change and agree on the first point. No need to execute code that is redundant.

Could you clarify the second point, though? What do you mean by altering the display of grid cells? When does this happen? I am not seeing anything wrong when I run `app.py` myself.

Reply:
Hey, sorry, I have been away for a bit, but I think we mean two different things when we say "alter" the outline of the map. What I meant to say is that if there is a double-digit number, which takes two places, the walls to the right will also be moved to the right by one, as seen in the image below, where I put the number 10 in one of the rows. You can see that the right outer wall was moved by one, which would also affect how the bot behaves.

One-digit numbers alter the map visually but not its outline, so it is totally up to you what you prefer! I only included the number of iterations as a number because I noticed my bot would visit certain cells multiple times and wanted to figure out why, but it is definitely not necessary. Ideally, there would be a setting "Show frequency of visited cells" that the user can toggle on/off when in debug mode, but I feel like this would overcomplicate the tool, which is trying to make testing easier in the first place. And, for most people, I think keeping the default '.' would be best. I hope I understood your comment correctly :)

![image](https://github.com/Celqaz/beatBruteBot/assets/19730248/0df37a17-b38b-4d7c-b08d-65979a2fc706)

Reply:
Hi @FedeCuci ,

Sorry for the late reply; I was just caught up in some personal stuff in the past weeks. 
After the brief review, I think there are no major issues in this PR. I will double-check in the coming week, and after we finalize the code, I will merge this PR. Cheers.